src/main/java/com/thindeck/dynamo/DyTxn.java

```diff
@@ -74,4 +101,29 @@ public void log(final String text) throws IOException {
     public Iterable<String> log() throws IOException {
         throw new UnsupportedOperationException("#log");
     }
+
+    /**
+     * Method that return the steps array attribute.
+     * @return The array of steps.
+     */
+    public Step[] getSteps() {
```

Comment:
`Dy` classes shouln't add more public methods, you have to find a way to test this without cheating :)